central/complianceoperator/v2/profiles/service/service_impl.go

```diff
@@ -79,26 +75,37 @@ func (s *serviceImpl) GetComplianceProfile(ctx context.Context, req *v2.Resource
 }
 
 // ListComplianceProfiles returns profiles matching given query
-func (s *serviceImpl) ListComplianceProfiles(ctx context.Context, query *v2.RawQuery) (*v2.ListComplianceProfilesResponse, error) {
-	// Fill in Query.
-	parsedQuery, err := search.ParseQuery(query.GetQuery(), search.MatchAllIfEmpty())
-	if err != nil {
-		return nil, errors.Wrapf(errox.InvalidArgs, "Unable to parse query %v", err)
+func (s *serviceImpl) ListComplianceProfiles(ctx context.Context, request *v2.ProfilesForClusterRequest) (*v2.ListComplianceProfilesResponse, error) {
```

Comment:
Strange - did this path change?

```
$ ./bin/roxcurl /v2/compliance/profiles
Not Found
```

But I was able to use:

```
 $ ./bin/roxcurl /v2/compliance/cluster/profiles
{"error":"cluster is required: invalid arguments","code":3,"message":"cluster is required: invalid arguments","details":[]}%
```

Reply:
Yes, that should be visible in the changes for `compliance_profile_service.proto`